src/org/opendatakit/briefcase/ui/reused/source/Source.java

```diff
@@ -190,6 +192,16 @@ static void clearAllPreferences(BriefcasePreferences prefs) {
    */
   String getDescription();
 
+  /**
+   * Returns pull instructions for forms
+   *
+   * @param forms {@link List} of forms to be pulled
+   * @return lines of instructions to pull
+   */
+  List<String> pullScriptLines(List<FormStatus> forms, BriefcasePreferences appPreferences);
+
+  List<String> pushScriptLines(List<FormStatus> forms, BriefcasePreferences appPreferences);
```

Comment:
So, return a `Map<String, String>` instead of this? Definitely not a list of strings with java invocations in them